src/NHibernate/Type/MetaType.cs

```diff
@@ -87,11 +86,6 @@ public override bool IsDirty(object old, object current, bool[] checkable, ISess
 			return checkable[0] && IsDirty(old, current, session);
 		}
 
-		public override object FromXMLNode(XmlNode xml, IMapping factory)
-		{
-			return FromXMLString(xml.Value, factory);
-		}
-
 		public object FromXMLString(string xml, IMapping factory)
```

Comment:
`FromXMLString` looks obsolete, no references found to it. A bit same for `ToXMLString`, only serves for `ToLoggableString` which could directly take `ToXmlString` implementation instead.